spec/views/email_subscriptions/unsubscribe.html.erb_spec.rb

```diff
@@ -4,6 +4,6 @@
   it "works" do
     assign(:email_type, "#{SiteConfig.community_name} digest emails")
     render
-    Approvals.verify(rendered, name: "email_subscriptions/unsubscribe", format: :html)
+    expect(rendered).to include("You have been unsubscribed from DEV(local) digest emails. 😔")
```

Comment:
should we avoid hardcoding `DEV(local)`?

Reply:
Good call! I'll use `SiteConfig.community_name` here instead 👍🏽